docs/overview/architecture-overview.md

```diff
@@ -154,29 +154,28 @@ Cross Origin Resource Sharing policies which prevent a browser page served at
 [https://example.com](https://example.com) from serving resources hosted at
 https://circleci.com.
 
-![CircleCi plugin talking to proxy talking to SaaS Circle CI](../assets/architecture-overview/circle-ci-plugin-architecture.png)
+![CircleCI plugin talking to proxy talking to SaaS Circle CI](../assets/architecture-overview/circle-ci-plugin-architecture.png)
 
 ## Databases
 
-As we have seen, both the lighthouse-audit-service and catalog-backend require a
+As we have seen, both the `lighthouse-audit-service` and `catalog-backend` require a
 database to work with.
 
-At the time of writing, the lighthouse-audit-service requires PostgreSQL to work
+At the time of writing, the `lighthouse-audit-service` requires PostgreSQL to work
 with. The service catalog backend uses an in-memory Sqlite3 instance. This is a
-development oriented setup and there are plans to support other databases in the
+development-oriented setup and there are plans to support other databases in the
 future.
 
-To learn more about the future of databases and Backstage, see the following two
+To learn more about the future of databases and Backstage, see the following
```

Comment:
Removed two so that the bullets below can be changed (added/removed) without adjusting this sentence